vue-shop.js

```diff
@@ -33,10 +33,10 @@ let layout = `
     <div class="pt-4 text-center">
         <div class="container">
             <div class="m-4 text-center" v-if="!Products.length">
-                <h3> No products found</h3>
+                <p class="h3"> No products found</p>
```

Comment:
Is this an intentional change? It's not as accessible as the heading tag is!

Reply:
It was a change I made quickly to match the other js files without thinking, not set in stone or anything though. I can update them all to use a heading tag instead.